solo-chains/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -91,3 +127,32 @@ impl snowbridge_pallet_ethereum_client::Config for Runtime {
     type ForkVersions = ChainForkVersions;
     type WeightInfo = weights::snowbridge_pallet_ethereum_client::SubstrateWeight<Runtime>;
 }
+
+impl snowbridge_pallet_system::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type OutboundQueue = EthereumOutboundQueue;
+    type SiblingOrigin = EnsureXcm<AllowSiblingsOnly>;
+    type AgentIdOf = snowbridge_core::AgentIdOf;
+    type TreasuryAccount = TreasuryAccount;
+    type Token = Balances;
+    type WeightInfo = ();
+    //type WeightInfo = crate::weights::snowbridge_pallet_system::WeightInfo<Runtime>;
+    #[cfg(feature = "runtime-benchmarks")]
+    type Helper = benchmark_helper::EthSystemBenchHelper;
+    type DefaultPricingParameters = Parameters;
+    type InboundDeliveryCost = ();
+    //type InboundDeliveryCost = EthereumInboundQueue;
```

Comment:
same thing with benchmarks

Reply:
I will do benchmarks when the PR is ready, as they are time consuming

Reply:
so can you do at least local benchmarks?

Reply:
ignore